test/integration/targets/filters/tasks/main.yml

```diff
@@ -16,6 +16,11 @@
 # You should have received a copy of the GNU General Public License
 # along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
 
+- name: install required libraries
+  pip:
+    requirements: "{{ role_path }}/files/requirements.txt"
+  become: True
+
```

Comment:
Filter requirements must be installed before ansible-playbook is started, so you'll want to install them here:

https://github.com/ansible/ansible/blob/f49469f7bf42297ec47de16e7a5e851fbf1f442b/test/integration/targets/filters/runme.sh#L7

Reply:
OK Cool